src/type-layout.md

```diff
@@ -274,28 +295,42 @@ For all other enumerations, the layout is unspecified.
 
 Likewise, combining two primitive representations together is unspecified.
 
-### The `align` Representation
+### The `align` modifier
 
-The `align` representation can be used on `struct`s and `union`s to raise the
+The `align` modifier can be used on `struct`s and `union`s to raise the
 alignment of the type to a given value.
 
-Alignment is specified as a parameter in the form of `#[repr(align(x))]`. The
-alignment value must be a power of two of type `u32`. The `align` representation
-can raise the alignment of a type to be greater than it's primitive alignment,
-it cannot lower the alignment of a type.
+The alignment is specified as an integer parameter in the form of
+`#[repr(align(x))]`. The alignment value must be a power of two from 1 up to
+2<sup>29</sup>.
+
+The `align` modifier raises the type's alignment to the specified alignment.
+If the specified alignment is less than the alignment of the type without the
```

Comment:
I believe this concept of "without the ** modifier" is referred to as the "natural **", for example "natural alignment". I think it would be good to introduce such terms earlier (when we first start to speak about alignment) and then use them.